src/assets/i18n/en.json

```diff
@@ -907,7 +907,7 @@
       "no-events-advice": "Welcome to My Events. It seems you don’t have any events. Let’s start to join event!",
       "join-event": "Join Event"
     },
-    "habits-in-progress": "in progress",
+    "habits-in-progress": "habits in progress",
```

Comment:
Need to be "In progress" - user story https://github.com/ita-social-projects/GreenCity/issues/988
![image](https://user-images.githubusercontent.com/104850911/233098128-9de61334-887e-4629-ae69-b66e297db2b4.png)

Reply:
corrected translation